internal/build/test_utils.go

```diff
@@ -137,7 +137,11 @@ func (f *dockerBuildFixture) startRegistry() {
 		f.t.Fatal(err)
 	}
 
-	err = stdout.WaitUntilContains("listening on", 5*time.Second)
+	if isCircleCI() {
+		err = stderr.WaitUntilContains("listening on", 5*time.Second)
```

Comment:
womp womp. but legit.

I think i'd prefer `combinedOut.WaitUntilContains` and then we wouldn't need this conditional -- any way to do that?

Reply:
Check it out now!